pkg/controller/plan/migration.go

```diff
@@ -1170,7 +1310,50 @@ func (r *Migration) waitingForFirstConsumer(vm *plan.VMStatus) (waiting bool, er
 	return
 }
 
-//
+func (r *Migration) updateCopyProgressForOvirt(vm *plan.VMStatus, step *plan.Step) (err error) {
+	if r.vmMap == nil {
+		r.vmMap, err = r.kubevirt.VirtualMachineMap()
+		if err != nil {
+			return
+		}
+	}
+
+	var vmCr VirtualMachine
+	found := false
+	if vmCr, found = r.vmMap[vm.ID]; !found {
+		msg := "VirtualMachine CR not found."
```

Comment:
how can this happen, when someone modifies the migration CR?

Reply:
I'm not sure to be honest, it's a copy paste, it may not be actually needed here

Reply:
currently, the code is based on the idea the VM posted, then it takes details from it. With the change in https://github.com/kubev2v/forklift/pull/48 we probably won't need it.
Here below we just search for the volumes, we can query them using the client.